census/models/question.js

```diff
@@ -76,7 +76,30 @@ module.exports = function(sequelize, DataTypes) {
       }
     ],
     instanceMethods: {
-      translated: mixins.translated
+      translated: mixins.translated,
+      scoreForAnswer: function(answer) {
+        /* Return the score for the provided answer. Either the full score or 0. */
+
+        let returnScore = (this.pass(answer)) ? this.score : 0;
+        return returnScore;
+      },
+      pass: function(answer) {
+        /* Determine whether the provided answer passes the question. */
+
+        // If config doesn't provide a way of passing, answer can't pass.
+        if (!_.has(this.config, 'score.passValue'))
+          return false;
+
+        let expected = _.get(this.config, 'score.passValue');
+        if (!_.isArray(expected)) {
+          expected = [expected];
+        }
```

Comment:
I'm not sure what's your use case here, but in case `expected` is not an array, this will not transform it into one, it will just *wrap* it into an array like this:

```javascript
> a = {a: 1, b: 2}
{ a: 1, b: 2 }
> [a]
[ { a: 1, b: 2 } ]
```